### ISSUE TITLE
Couchbase Lite 1.x: Update REST POST _replicate API

### ISSUE THREAD

**Issue by rajagp** [couchbaselabs/couchbase-mobile-portal#815]
According to forum post, https://forums.couchbase.com/t/how-to-sync-a-specific-channel-with-ionic2-couchbase-lite-2-0-and-sync-gateway-1-5/14300/4 , the following needs to be specified for filtering on channels
"In the POST to start the replication, add the JSON properties: "filter":"sync_gateway/bychannel", "channels":"ABC,NBC,CNN,FOX"

The current documentation needs to be updated accordingly https://developer.couchbase.com/documentation/mobile/1.4/references/couchbase-lite/rest-api/index.html?v=1.4#/server/post__replicate

**Comment by snej** [comment]
Hm. The existing docs don't really document the parameters to `_replicate` at all; they just give an "example" containing a ton of parameters without describing any of them. This is really inadequate.

The CouchDB documentation has more info. But last I checked it was incomplete too. I added a bunch of documentation to the old CouchDB wiki when I was reverse-engineering the details to implement the original TouchDB replicator, but I don't know if it still exists.
